src/task.coffee

```diff
@@ -82,8 +82,8 @@ _run = async (ran, name, args) ->
       else
         yield f args...
 
-    finish = Date.now()
-    console.log "Task '#{name}' completed in #{finish - start}ms."
+    # finish = Date.now()
+    # console.log "Task '#{name}' completed in #{finish - start}ms."
```

Comment:
Actually, I don't think was a debugging statement, I think it was a desire to add task completion times. But for some reason I think I just moved them somewhere else and forgot to remove them here.